jquery.swiftype.autocomplete.js

```diff
@@ -119,6 +118,16 @@
         }
       };
 
+      $this.showNoResults = function () {
+        $list.empty();
+        if (config.noResultsMessage !== undefined) {
```

Comment:
maybe switch this to be a positive check

Reply:
fixed